README.md

```diff
@@ -1,20 +1,13 @@
-[![Travis Status][trav_img]][trav_site]
-
 Victory
 =======
-Victory is an opinionated set of data visualization tools built in React. This repo aggregates all of the stable Victory components so they can be conveniently included. 
-
-IMPORTANT
```

Comment:
Please leave in some kind of warning about the state of the project. Maybe call it an alpha instead of a pre-release, but with the same caveats about a somewhat unstable api